RELEASE.md

```diff
@@ -24,18 +26,20 @@ Finally, on the proposed due date the assignees for the upcoming release proceed
 
 ### 3. Release PR
 
+- From the `master` branch checkout a new branch titled the release number such as `git checkout -b 0.26.0`
```

Comment:
```suggestion
- From the `master` branch checkout a new branch titled the release number such as `git checkout -b x.y.z`
```

Anyways, do we want to enforce the branch name?
Once merged that branch will be automatically deleted...